dll/win32/shell32/CShellDispatch.cpp

```diff
@@ -119,58 +128,83 @@ HRESULT STDMETHODCALLTYPE CShellDispatch::BrowseForFolder(LONG Hwnd, BSTR Title,
 HRESULT STDMETHODCALLTYPE CShellDispatch::Windows(IDispatch **ppid)
 {
     TRACE("(%p, %p)\n", this, ppid);
+    return CoCreateInstance(CLSID_ShellWindows, NULL, CLSCTX_LOCAL_SERVER, IID_PPV_ARG(IDispatch, ppid));
+}
 
-    *ppid = NULL;
+static HRESULT SHELL_OpenFolder(LPCITEMIDLIST pidl, LPCWSTR Verb = NULL)
+{
+    // Custom function because SHOpenFolderAndSelectItems does not support non-FS pidls nor verbs.
```

Comment:
Is it a known limitation of that SHOpen* function? (just to know whether it is, or whether it's a ROS-only bug)